Makefile.am

```diff
@@ -3,11 +3,13 @@ dist_bin_SCRIPTS = \
 	create-update.sh \
 	build-chroots.sh \
 	init-mix.sh \
-	init-versions.sh \
```

Comment:
I'm okay with that approach. The removal was something that glaringly stood out when reviewing this patch, as there was no mention of it in the commit message.